### ISSUE TITLE
Just a friendly reminder of the new materialize repo

### ISSUE THREAD

**Issue by xenkuo** [Dogfalo/materialize#6601]
I've waited any real update of this repo for a quite long time, but nothing except the donation kind of update. You may have the same question will me.
Now there is a new fork to this repo, which I believe it could be call real materialize css repo:
https://github.com/materializecss/materialize

You may also refer https://github.com/Dogfalo/materialize/issues/6438 to get a whole view about this change.

And also I wanna thanks those guys who create and maintain that repo.

**Comment by xenkuo** [comment]
By the way, there will be a new release in recent days according with the discussions.
What a wonderful day when I got this info.

**Comment by j3g** [comment]
yes, the forked project is dusting off the dirt and getting the project back to 2020. more progress there than in the last year in this repo.
the maintainers here, we thank you for the hard work. and now that you are gone, please hand over the project properly.
